modules/osd-create-cluster-ccs.adoc

```diff
@@ -220,6 +220,11 @@ CIDR configurations cannot be changed later. Confirm your selections with your n
 You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
 ====
 +
+.. Provide administrator approval per your cluster update method:
+** Individual updates: If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
+** Recurring updates: If you selected recurring updates for your cluster, provide an administrator’s acknowledgment and click *Approve and continue*. {cluster-manager} does not start scheduled y-stream updates for minor versions without receiving an administrator’s acknowledgment.
++
+For information about administrator acknowledgment, see link:https://docs.openshift.com/dedicated/upgrading/osd-upgrading-cluster-prepare.html#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
```

Comment:
Are allowed to use links in modules now?

Reply:
Updated to xref. Apologies for the incorrect crossreferencing.